pages.pt_BR/windows/choco-install.md

```diff
@@ -0,0 +1,36 @@
+# choco install
+
+> Instalar um pacote ou mais com Chocolatey.
+> Mais informação: <https://chocolatey.org/docs/commands-install>.
+
+- Instalar um ou mais pacotes separado por espaço:
+
+`choco install {{package(s)}}`
+
+- Instalar pacotes a partir de um aquivo de configuração personalizado:
+
+`choco install {{path/to/packages.config}}`
+
+- Instalar um arquivo específico "nuspec" ou "nupkg":
+
+`choco install {{path/to/file}}`
+
+- Instalar uma versão específica de um pacote:
+
+`choco install {{package}} --version {{version}}`
```

Comment:
Remember that you have to translate the arguments to make this easy to read for non-English speakers. e.g. `package` -> `pacote`